### PR TITLE
COMPAT: listParts returns <ETag> section with quote

### LINKED ISSUE #315: COMPAT: AWS listParts returns <ETag> section with quote
It looks like when we do `listParts` (MPU) we respond an XML with a <ETag> section without quode. 
However, AWS responds an XML with a `<ETag>` section with quote.

**S3SERVER:** `<ETag>93b885adfe0da089cdf634904fd59f71</ETag>`
**AWS:** `<ETag>"93b885adfe0da089cdf634904fd59f71"</ETag>`

On **ceph/s3-tests**, these changes fix:
`ERROR: s3tests.functional.test_s3.test_multipart_upload_incorrect_etag`

### PR DESCRIPTION
**Pull request by nicolas2bert**
FIXES #315

### CODE REVIEW

File: tests/functional/aws-node-sdk/test/legacy/tests.js [hunk 1]
```diff
@@ -254,11 +258,11 @@ describe('aws-node-sdk test suite as registered user', function testSuite() {
             MultipartUpload: {
                 Parts: [
                     {
-                        ETag: `"${calculatedFirstPartHash}"`,
+                        ETag: calculatedFirstPartHash,
```
Comment (bennettbuchanan) [inline tests/functional/aws-node-sdk/test/legacy/tests.js]: Won't these `ETag` values of an MPU JSON structure need to remain in quotes?

File: lib/api/listParts.js [hunk 2]
```diff
@@ -206,7 +206,7 @@ export default function listParts(authInfo, request, log, callback) {
                 buildXML([
                     { tag: 'PartNumber', value: part.partNumber },
                     { tag: 'LastModified', value: part.lastModified },
-                    { tag: 'ETag', value: part.ETag },
+                    { tag: 'ETag', value: `"${part.ETag}"` },
```
Comment (bennettbuchanan) [inline lib/api/listParts.js]: Nice catch 👍

File: tests/functional/aws-node-sdk/test/legacy/tests.js (hunk 1 above)
Comment (bennettbuchanan) [inline tests/functional/aws-node-sdk/test/legacy/tests.js]: As per our conversation, indeed these quotes are unnecessary to create the MPU JSON structure.

**Comment by nicolas2bert** [comment]
@ironman-machine go

**Pull request merged**